content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

```diff
@@ -67,6 +67,24 @@ This status means that logs are enabled but you haven't specified which containe
 
 2. To configure the Agent to collect from other containers, set the `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` environment variable to `true`.
 
+
+## Docker log collection from file issues
+
+When collecting Docker container logs from file the Agent will fall back on collection from Docker socket if it cannot read from the directory where Docker container logs are stored (`/var/lib/docker/containers` on Linux). In some circumstances the Datadog agent may fail to collect log from file. This can be diagnosed by checking the Logs Agent status and looking for an file type entry showing an error like:
+
+```text
+    - Type: file
+      Identifier: ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834
+      Path: /var/lib/docker/containers/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834-json.log
+      Status: Error: file /var/lib/docker/containers/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834-json.log does not exist
+```
+
+This status means that the agent is unable to find a log file for a given container, to resolve this issue check that the folder containing Docker containers logs is correctly exposed to the Datadog Agent container. On Linux it corresponds to  `-v /var/lib/docker/containers:/var/lib/docker/containers:ro` on the command line starting the Agent container whereas on Windows it translates to `-v c:/programdata/docker/containers:c:/programdata/docker/containers:ro`. Note that the directory relatively to the underlying host may be different due to spectific config of the Docker daemon, this is not an issue pending a correct Docker volume mapping. E.g. `-v /data/docker/containers:/var/lib/docker/containers:ro` if the Docker data directory has been relocated to `/data/docker` on the underlying host.
+
+If log are collected but single appears to be split please check that the Docker daemon is using the [JSON logging driver][#your-containers-are-not-using-the-json-logging-driver].
```

Comment:
```suggestion
If logs are collected but single lines appear to be split, check that the Docker daemon is using the [JSON logging driver][#your-containers-are-not-using-the-json-logging-driver].
```
@prognant I've assumed that you're talking about split lines here, could you verify?

Reply:
I think you got it. The issue I want to highlight would be :
* Customer app emit a (long > 16kB) line : `A sample log line`
* In the log explorer the customer sees two (or more) record that corresponds to piece of the initial line:
    *  `A sample`
    * `log line`